posts/models.py

```diff
@@ -1,4 +1,5 @@
 import os
+from tarfile import StreamError
```

Comment:
Do we need this using?

Reply:
Good catch, this must be vscode doing its thing, I'll take this out. Thanks!